mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -217,8 +218,8 @@ public PreDeletionHooks getPreDeletionHooks() {
     /**
      * Generate and return the next uid validity
      */
-    protected int randomUidValidity() {
-        return Math.abs(RANDOM.nextInt());
+    protected UidValidity randomUidValidity() {
+        return UidValidity.of(Math.abs(RANDOM.nextInt()));
```

Comment:
Won't work, nextInt could be negative.

Reply:
there's an abs.

However:
 - The unlikely `0` edge case is an issue
 - We use half of the possible values hence we increase conflict risks...